src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java

```diff
@@ -158,14 +159,16 @@ private void initialize() {
             viewModel = new ManageStudyDefinitionViewModel(
                     prefs.getImportFormatPreferences(),
                     prefs.getImporterPreferences(),
-                    dialogService);
+                    dialogService,
+                    prefs);
```

Comment:
Two lines up there are prefs injected in the detailed prefs importformatprefs and importerprefs, here you inject the whole prefs class. Keep the abstraction level!

Reply:
Make a method `getCatalogPrferences` and use that, you mean?

Reply:
No, I only see that you are using WorkspacePreferences in your changes, so inject them, not the whole PreferencesService